gatsby-node.js

```diff
@@ -4,13 +4,36 @@ const createPeople = require("./src/create-pages/createPeople");
 const createRanking = require("./src/create-pages/createRanking");
 const createIntlPages = require("./src/create-pages/intl");
 
-exports.onCreateWebpackConfig = ({ stage, actions }) => {
-  actions.setWebpackConfig({
+exports.onCreateWebpackConfig = ({ actions, getConfig }) => {
+  const config = getConfig();
+  const { module, resolve } = config;
+
+  actions.replaceWebpackConfig({
+    ...config,
+
     resolve: {
-      modules: [
-        // path.resolve(__dirname, "src"),
-        __dirname,
-        "node_modules",
+      ...resolve,
+      modules: [__dirname, "node_modules"],
+    },
+
+    module: {
+      ...module,
+      rules: [
+        // https://github.com/Negan1911/storybook-svgr-react-component/blob/master/index.js
+        ...module.rules.map((_) => {
+          if (_.test?.toString().includes("svg|")) {
+            return {
+              ..._,
+              test: new RegExp(_.test.source.replace("svg|", "")),
+            };
```

Comment:
Can we resolve this problem by put svg rule at first position of rules?

Reply:
Good thinking. I've tried, but unfortunately it's not working..
<img width="1193" alt="Screen Shot 2021-04-23 at 8 47 29 PM" src="https://user-images.githubusercontent.com/123074/115845676-33ac7080-a475-11eb-899b-2e4464076fbd.png">